src/extension.ts

```diff
@@ -253,6 +253,40 @@ export const checkConnection = (clearCookies = false): void => {
     });
 };
 
+async function serverManager(): Promise<void> {
+  const extId = "intersystems-community.servermanager";
+  const ignore =
+    config("ignoreInstallServerManager") ||
+    vscode.workspace.getConfiguration("intersystems.servers").get("/ignore", false);
+  if (ignore || vscode.extensions.getExtension(extId)) {
+    return;
+  }
+  return vscode.window
+    .showInformationMessage(
+      `Extension InterSystems® Server Manager have not installed,
+      it's recommended to install it.`,
+      "Install",
+      "Skip",
+      "Ignore"
+    )
+    .then(async (action) => {
+      switch (action) {
+        case "Install":
+          await vscode.commands.executeCommand("workbench.extensions.installExtension", extId);
+        // eslint-disable-next-line no-fallthrough
+        case "View":
+          await vscode.commands.executeCommand("workbench.extensions.search", `@tag:"intersystems"`);
```

Comment:
Why not simply search for extId?

Reply:
search for extId just will show only one extension on the left panel, so, I decided to show any intersystems tagged extensions, but open particular extension in full view.

Reply:
Since there's no direct path to `case "View":` the user picks "Install", the servermanager gets installed, then you show them all intersystems-tagged extensions in the list on the left, and the servermanager one displays in detail on the right. Is that what you mean? Seems a bit unexpected, but I can live with it.